.gitignore

```diff
@@ -39,3 +39,9 @@ _testmain.go
 # code coverage
 coverage.html
 cover.out
+
+# fuzzing
+test/fuzz-reader/corpus
+test/fuzz-reader/crashes
+test/fuzz-reader/suppressions
+test/fuzz-reader/*.zip
```

Comment:
These files just bloat the repo. We can share them between ourselves (or from the community) if needed. Keeping them around helps later runs too.